.github/workflows/deploy-staging.yml

```diff
@@ -29,7 +29,9 @@ jobs:
       - name: Install dependencies
         run: yarn install
       - name: Build packages and storybook
-        run: yarn build
+        run: |
+          yarn build
```

Comment:
Can you change this to only build our storybooks and not the entire project `build` script?